0.2_hyperflow/0.2.2_flow_cycles.py

```diff
@@ -0,0 +1,92 @@
+### flow query on a loaded dg 
+## the goal is to search for a specific path through the network
+
+# include the grammar used to create the dg 
+include("grammar_formose.py")
+
+
+# define the name of the dg to load
+dg_name = "dg_autocat"
+# this dg has been created before for this purpose, the script to make it is called make_autocat_dg.py
+
+# load the dg in question from the previous dg dump file
+dg = DG.load(inputGraphs, inputRules, f"{dg_name}.dg") 
+
+
+# explore the size of the dg
+def display_size_dg(dg):
+    print("============ Number vertices ============")
+    print(dg.numVertices)
+    print("============ No edges ============")
+    print(dg.numEdges)
+
+display_size_dg(dg)
+
+### set up a flow model 
+# the flow model is the model we create that consists of linear equations to feed into an ILP solver
+# give the flow a name to keep an overview
+flow_name = "flow_cycle"
+
+
+## this is the actual flow model ##
+flow = hyperflow.Model(dg, ilpSolver = "CPLEX") # specified to use the CPLEX ILP solver
+# the default solver is Gurobi, if you don't have an academic license for that use CPLEX

```

Comment:
Not correct: if Gurobi is not compiled in, then it is not the default solver.
I suggest to leave out the ``ilpSolver = "CPLEX"``